slingr-stack/github-app-endpoint
# #13389 standardize GitHub endpoint

Pull request for the topic https://github.com/slingr-stack/github-app-endpoint/issues/3 created by @pasaperez-slingr 

## What does it do?

This ticket implements the use of slingr-helpgen, whereby it modifies existing functions, adds basic functions and step flows.

## How to test it?

Make calls from the platform for example using the console using the information provided in the documentation. Or you can use the helpers or flow steps in an action and trigger it from the runtime. Make sure to previously update to version "v0.267.0-dev" in the developer portal and in the application builder. 

## Implementation notes

Create new version of the endpoint, it is not compatible with previous versions.

## Implementation notes

Create a new tags changing the major version and deploy to the platform.
